{"uuid":"60c284e3-4ee3-4555-8c7c-dae2f99b9041","historyId":"57f378c765e8d85e88bc61db97a53f8d","fullName":"testcases.TestLoginPage.TestUserShouldNotLoginWithInvalidUserCredentials","labels":[{"name":"package","value":"testcases.TestLoginPage"},{"name":"testClass","value":"testcases.TestLoginPage"},{"name":"testMethod","value":"TestUserShouldNotLoginWithInvalidUserCredentials"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"WebAutomationWiki"},{"name":"subSuite","value":"testcases.TestLoginPage"},{"name":"host","value":"LAPTOP-TSSPRPJT"},{"name":"thread","value":"12136@LAPTOP-TSSPRPJT.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"TestUserShouldNotLoginWithInvalidUserCredentials","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [There are problems with some of your input.] but found [Incorrect username or password entered. Please try again.]","trace":"java.lang.AssertionError: expected [There are problems with some of your input.] but found [Incorrect username or password entered. Please try again.]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat testcases.TestLoginPage.TestUserShouldNotLoginWithInvalidUserCredentials(TestLoginPage.java:70)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg2","value":"There are problems with some of your input."},{"name":"arg1","value":"Password"},{"name":"arg0","value":"Username"}],"start":1750546321062,"stop":1750546326902}